helpers/dune

```diff
@@ -1,5 +1,5 @@
 (library
  (name helpers)
- (libraries lwt uri zarith hex digestif)
+ (libraries lwt lwt.unix uri zarith hex digestif)
```

Comment:
This change is not needed

Reply:
Indeed. Left over from the previous change that moved the file io to helpers.